xml/hwbp_registers.xml

```diff
@@ -987,7 +996,16 @@
             If the H extension is not supported, the only legal values are 0 and 4.
         </field>
 
-        <field name="0" bits="22:18" access="R" reset="0" />
+        <field name="0" bits="22:20" access="R" reset="0" />
+        
+        <field name="sbytemask" bits="19:18" access="WARL" reset="0">
+            0=compare, 1=don't compare byte, i.e. byte is masked out
+            bit 18 = lower byte compare of svalue (bits [7:0])
+            bit 19 = upper byte compare of svalue (bits [15:8])
+            
+            Byte masking of svalue only applies to scontext comparison; i.e.
+            when sselect==1.
```

Comment:
I think I mentioned that adequately in my suggestion enough. If not, use \FcsrTextraThirtytwoSselect=1 instead of sselect==1.